assignments/prototypes.js

```diff
@@ -14,15 +14,33 @@
   * dimensions
   * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
 */
-
+function GameObject(characterAttributes)
+{
+  this.createdAt = characterAttributes.createdAt;
+  this.dimensions = characterAttributes.dimensions;
+}
+GameObject.prototype.destroy = function() 
+{
+return  `${this.name} was removed from the game.`;
+}
 /*
   === CharacterStats ===
   * hp
   * name
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
-
+function CharacterStats(characterAttributes)
+{
+  GameObject.call (this, characterAttributes);
+  this.hp = characterAttributes.hp;
+  this.name = characterAttributes.name;
+}
+CharacterStats.prototype = Object.create(GameObject.prototype);
+CharacterStats.prototype.takeDamage = function()
+{
+  return `${this.name} took damage.`;
+}
```

Comment:
Once again, your code is good, but maybe just clean up the placing of your curly braces and remember semicolon on line 43.